site/cypress/e2e/Cypress-Gradeable/hotkey.spec.js

```diff
@@ -0,0 +1,57 @@
+describe('TA grading hotkey testing', () => {
+    after(() => {
+        cy.login();
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'update?nav_tab=2']);
+        cy.get('[data-testid="peer-component-container"]').should('exist');
```

Comment:
I don't think this is the best way to do this, while this may do it correctly in an ideal world, it will still leave peer components if the test is re-run multiple times without hitting the after block. I am going to tag @williamjallen to see if he thinks this is good enough, (as it will work in CI, but may cause problems to local users). 

At the very least, you can test that after you click the 'delete gradeable component' button, assert that there are no more peer-components left.

Reply:
What I think why the CI passed , as there is no peer component and then i add one peer component and later i deleted that one , so i am adding one line for assertion that no more peer component left 

later i will change if i get any other way to do this

Reply:
After discussing with William, I think this is fine for now, just add the assertion that the peer component doesn't exist, and I'll create an issue to refactor some of the older tests that require the state of the VM to be in a certain way, to use before blocks to set the state.